diffsims/generators/sphere_mesh_generators.py

```diff
@@ -442,6 +442,32 @@ def get_icosahedral_mesh_vertices(resolution):
     return vertices
 
 
+def get_random_sphere_vertices(resolution):
+    """
+    Create a mesh that randomly samples the surface of a sphere
+
+    Parameters
+    ----------
+    resolution: float
+        The expected mean angle between nearest neighbor
+        grid points in degrees.
+
+    Returns
+    -------
+    points_in_cartesian : numpy.ndarray (N,3)
+        Rows are x, y, z where z is the 001 pole direction
+
+    References
+    ----------
+    https://mathworld.wolfram.com/SpherePointPicking.html
+    """
+    # convert resolution in degrees to number of points
+    number = int(1/(4*np.pi)*(360/resolution)**2)
+    xyz = np.random.normal(size=(number, 3))
```

Comment:
Could we change this to follow the new numpy style (see [1]), and having done so other `seed` as a kwarg?

[1] https://numpy.org/doc/stable/reference/random/index.html?highlight=numpy%20random#module-numpy.random